src/index.js

```diff
@@ -44,7 +45,10 @@ export default class KintoAdmin extends Component<Props> {
       this.store.dispatch(
         sessionActions.getServerInfo({
           authType: "anonymous",
-          server: DEFAULT_KINTO_SERVER,
+          server:
+            settings.singleServer ||
+            (history && history[0]) ||
```

Comment:
Should we have a test for that too?

Reply:
I'm just a bit puzzle that it looks very similar to https://github.com/Kinto/kinto-admin/blob/master/src/components/AuthForm.js#L315
I know that we don't want to refactor it now, but can we at least add a comment to «connect» the two places for future contributions?

Reply:
I've just refactored it in `utils/getServerByPriority`